index.ts

```diff
@@ -12,7 +12,7 @@ addTests('__urls_that_dont_match__', [
 	'https://github.com/sindresorhus/refined-github/issues/templates/edit',
 ]);
 
-export const is404 = (): boolean => document.title.startsWith('Page not found · GitHub'); // #98
+export const is404 = (): boolean => document.title.startsWith('Page not found · GitHub') || document.title.startsWith('File not found · GitHub'); // #98
```

Comment:
In this case it's best to just use regex. I think this:

```suggestion
export const is404 = (): boolean => /^(Page|File) not found · GitHub/.test(document.title); // #98
```